build.gradle

```diff
@@ -44,20 +44,19 @@ ext {
     lombokVersion = '1.18.2'
     springSwaggerVersion = '2.9.2'
     commonsLangVersion = '3.8'
-//    log4jVersion = '2.11.1'
-//    slf4jVersion = '1.7.25'
     aspectjVersion = '1.9.1'
     spongycastleVersion = '1.58.0.0'
     mockitoCoreVersion = '2.22.0'
     hibernateTypesVersion = '2.3.0'
     testContainersVersion = '1.8.3'
-    postgresVersion = '42.2.4'
```

Comment:
why did you remove postgres?

Reply:
it is managed by spring-boot-gradle-plugin